FlagsmithClient/src/main/java/com/flagsmith/entities/Flag.kt

```diff
@@ -1,17 +1,6 @@
 package com.flagsmith.entities
 
-import com.flagsmith.internal.Deserializer
-import com.flagsmith.internal.fromJson
 import com.google.gson.annotations.SerializedName
-import com.google.gson.reflect.TypeToken
-import java.io.Reader
-
-class FlagListDeserializer: Deserializer<List<Flag>> {
```

Comment:
I guess this was removed in the move from Fuel -> Retrofit?

Reply:
Yea no need for it any more, though some of this code lives on in the tests to ease things